### DISCUSSION TITLE
appid的正确格式是什么？

### DISCUSSION THREAD

**Discussion by fredericky123** [SeaHOH/GotoX#100] [问答/咨询]
RT,输入后总是提示错误，请问正确格式是啥

**Comment by fredericky123** [comment]
能否给一个配置好的config，用默认的tls一个ip都不行，大家是用什么模式

**Comment by SeaHOH** [comment]
直接填 ID 就好，多个用 `|` 分隔。
我尝试下来是没几个可用 IP 的，所以不推荐使用 GAE 代理功能。
不需要配置 ID 就能直接访问谷歌。

**Comment by shpherd** [comment]
我靠，啥都没改就连上油管了。之前安装了vc_redist.x64.exe（VC++2015）和Vcruntime140.dll。
能上www.google.com和邮箱，不能上IPV6.GOOGLE.COM，TWITTER.COM和https://www.facebook.com/不能上。
提示AppID 配置为空，无法使用 GAE 代理
请编辑 'D:\\64bit software\\chrome plugins\\GotoX-3.5.0-cp35-win32\\config\\Config.ini' 文件，添加可用的 AppID 到 [gae] 配置中并重启 GotoX！
但是config.ini一旦修改就不扫了，报程序错。

**Comment by SeaHOH** [comment]
提交错误日志我才能帮到你。


> 不能上IPV6.GOOGLE.COM，TWITTER.COM和https://www.facebook.com/ 不能上。

访问谷歌没必要使用特别的域名，那需要配置规则适配限制，而且和普通域名提供的功能没有区别。
twitter 需要代理。
FB 访问需要 IPv6 + 伪造证书，自动规则配置有说明和示例。

**Comment by shpherd** [comment]
traceback <most recent call last>:
file "..laucher\win32.py", line 386, in  <module>
       start_gotox<>
file "..laucher\win32.py", line 185, in  start_gotox
       load_config<>
file "..laucher\win32.py", line 42, in   load_config
       config.read<[config_filename,config_user_filename]>
file "..laucher\win32.py", line 32, in   <lambda>
       configparser.read = lambda s, f , encoding='utf8': _read<s, f, encoding>
file "configparser.py", line  696,  in read
file "configparser.py", line  1077,  in_read
configparser.missingsectionheadererror:   file contains no section headers.
file:'  D:\\64bit software\\chrome plugins\\GotoX-3.5.0-cp35-win32\\config\\config.ini'.  line 1
'\ufeff#coding:utf-8\n'

以上手敲，奇怪不能上传截图。
使用gotox生成桌面快捷方式"D:\64bit software\chrome plugins\GotoX-3.5.0-cp35-win32\python\python.exe" ..\launcher\win32.py  双击运行。
config.ini添加ID，这些ID以前在XXNET中使用。
我觉得不需要r6上传，这些ID已经在用。

**Comment by SeaHOH** [comment]
日志可以直接复制。
出错是因为使用记事本编辑，请不要使用记事本。
服务端和 XX-Net 并不兼容。

**Comment by shpherd** [comment]
notepad++搞定ID ，现在可以上twitter和instagram了。
点不动CMD的东西，右键出不来菜单，没法标记。
浏览器代理不用switchyomega插件，好像不用卸掉？就用gotox的自动代理就行。上次有提示8086和8087为监视端口。
别的东西需要再学习

**Comment by SeaHOH** [comment]
> 点不动CMD的东西，右键出不来菜单，没法标记。

直接左键选择，然后点击右键就已经复制完毕。
